bin/hardening/record_mac_edit.sh

```diff
@@ -17,64 +17,48 @@ HARDENING_LEVEL=4
 # shellcheck disable=2034
 DESCRIPTION="Record events that modify the system's mandatory access controls (MAC)."
 
-AUDIT_PARAMS='-w /etc/selinux/ -p wa -k MAC-policy'
-FILES_TO_SEARCH='/etc/audit/audit.rules /etc/audit/rules.d/audit.rules'
-FILE='/etc/audit/rules.d/audit.rules'
+AUDIT_PARAMS=("-w /etc/apparmor/ -p wa -k MAC-policy" "-w /etc/apparmor.d/ -p wa -k MAC-policy")
+AUDIT_FILE='/etc/audit/audit.rules'
+ADDITIONAL_PATH="/etc/audit/rules.d"
+FILE_TO_WRITE='/etc/audit/rules.d/audit.rules'
 
 # This function will be called if the script status is on enabled / audit mode
 audit() {
-    # define custom IFS and save default one
-    d_IFS=$IFS
-    c_IFS=$'\n'
-    IFS=$c_IFS
-    for AUDIT_VALUE in $AUDIT_PARAMS; do
-        debug "$AUDIT_VALUE should be in file $FILES_TO_SEARCH"
-        IFS=$d_IFS
+    MISSING_PARAMS=()
+    index=0
+    # use find here in order to simplify test usage with sudo using secaudit user
+    FILES_TO_SEARCH="$(sudo_wrapper find $ADDITIONAL_PATH -name '*.rules' | paste -s) $AUDIT_FILE"
```

Comment:
blocker : add coreutils in debian package dependencies

Reply:
done